PROTOCOL.md

```diff
@@ -326,6 +340,31 @@ The following is an example `remove` action.
 }
 ```
 
+### Add CDC File
+The `cdc` action is used to add a [file](#change-data-files) containing only the data that was changed as part of the transaction.
+
+The schema of the `cdc` action is as follows:
+
+Field Name | Data Type | Description
+-|-|-
+path| String | A relative path to a file from the root of the table or an absolute path to a file that should be removed from the table. The path is a URI as specified by [RFC 2396 URI Generic Syntax](https://www.ietf.org/rfc/rfc2396.txt), which needs to be decoded to get the file path.
```

Comment:
A relative path to a change data file from the root of the table or an absolute path to a change data file that should be added to the table